### ISSUE TITLE
A Question about TCGA-CRC Data Splits

### ISSUE THREAD

**Issue by beidiz** [mahmoodlab/PANTHER#11]
For the data splits of TCGA-CRC, the fold 0 has around 150 samples, while fold 1, 2, 3, 4 have only around 100 samples. Could you please check whether there are some problems with the uploaded data splits?

**Comment by andrewsong90** [comment]
Hi @mayonezu216 

The train/test split, as mentioned in the paper, is based on site stratification rather than exactly 80/20% splits on the patient-level. This is to prevent WSIs from a same tissue processing site ending up both in training and test dataset (data leakage). Since we are splitting based on sites, which might have variable number of patients, some fold might have larger test set than other folds. But that means the fold would have smaller training set.

Best,

**Comment by beidiz** [comment]
> Hi @mayonezu216
> 
> The train/test split, as mentioned in the paper, is based on site stratification rather than exactly 80/20% splits on the patient-level. This is to prevent WSIs from a same tissue processing site ending up both in training and test dataset (data leakage). Since we are splitting based on sites, which might have variable number of patients, some fold might have larger test set than other folds. But that means the fold would have smaller training set.
> 
> Best,

Thanks for the explanation!
